src/rvsdg/mod.rs

```diff
@@ -92,17 +92,17 @@ pub(crate) type Id = usize;
 #[derive(Debug, PartialEq)]
 pub(crate) enum Expr<Op> {
```

Comment:
optional: should we name the fields now that there are so many (similar to the various `Ops` in the bril Rust bindings?)

Reply:
We probably should (I wish there was a cool vscode refactor for this)